### DISCUSSION TITLE
Hyprland-git and Glava

### DISCUSSION THREAD

**Discussion by akashgagda** [prasanthrangan/hyprdots#970] [Q&A] [answered]
i have update hyprland to hyprland-git for using Glava also change new mouse name variable but still i'm getting error and Glava open in terminal not in background 
![image](https://github.com/prasanthrangan/hyprdots/assets/55386892/a96a4452-5728-4d22-95b2-1ed864b7b640)
![image](https://github.com/prasanthrangan/hyprdots/assets/55386892/a6c3318d-ef23-403d-8f87-62f9da1826f6)
![image](https://github.com/prasanthrangan/hyprdots/assets/55386892/ec1161f0-ff48-4705-9160-d21e30a16542)
![image](https://github.com/prasanthrangan/hyprdots/assets/55386892/007015fa-f77a-4b87-8127-c2dc3f97a913)
![image](https://github.com/prasanthrangan/hyprdots/assets/55386892/b6ca5f73-b08d-4d1f-b504-44e82b8447a3)

**Comment by kRHYME7** [comment, marked as answer]
https://github.com/prasanthrangan/hyprdots/discussions/252#discussioncomment-8594317 

You should be aware of possible breaking changes when using hyprland-git. Here's the patch: 
https://github.com/prasanthrangan/hyprdots/pull/930

**Reply by akashgagda** [reply]
plugin:hyprwinwrap:class = GLava

works for me after run hyprpm update

thank you bro
